config/database.yml

```diff
@@ -7,10 +7,13 @@ development:
   <<: *default
   database: unify_development
 
-test:
+test: &test
   <<: *default
   database: unify_test
 
 production:
   <<: *default
   database: unify_production
+
+cucumber:
+  <<: *test
```

Comment:
Why do you want/need to create a separate db for cucumber? Isn't the test db enough?